x-pack/platform/test/api_integration/apis/security/privileges.ts

```diff
@@ -138,6 +139,7 @@ export default function ({ getService }: FtrProviderContext) {
         'endpoint_list_read',
         'workflow_insights_all',
         'workflow_insights_read',
+        'global_artifact_management_all',
```

Comment:
__question:__ just for my understanding - how (conceptually) does space awareness in Fleet and the Security solution relate to the availability of the Global Artifacts management feature privilege?